qbatch

```diff
@@ -170,4 +202,8 @@ if __name__ == "__main__":
         os.chmod(script, os.stat(script).st_mode | stat.S_IXUSR)
         if dry_run:
             continue
-        subprocess.call(['qsub', script])
+        if system == 'sge' or system == 'pbs':
+            subprocess.call(['qsub', script])
+        else:
+            print "Launching Local Jobscript, saving output to logdir/{}.log".format(job_name)
```

Comment:
Perhaps a `--logdir` or `--logfile` option for where to stuff log files?

I'm also thinking, no logging to a file to save us the complexity. Let the caller/scheduling system handle that. This would also clean up our subprocess handling code.

Thoughts?